doc/classes/Skeleton3D.xml

```diff
@@ -184,6 +206,10 @@
 			<argument index="1" name="custom_pose" type="Transform">
 			</argument>
 			<description>
+				Sets the custom pose transform, [code]custom_pose[/code], for the bone at [code]bone_idx[/code].
+
+				[b]Note[/b]: The pose transform needs to be in bone space. Use [method world_transform_to_bone_transform]
+				to convert a Spatial transform to bone space.
```

Comment:
This one is a global transform right?

A spatial is relation to the root node. While the bone is in relation to the skeleton or just the bone.

Reply:
Or did you write spatial as a shorthand for Transform3D.

Reply:
The custom pose is a replacement for the bone rest.